pkg/policy/mapstate.go

```diff
@@ -147,6 +152,29 @@ func (e *MapStateEntry) RemoveDependent(key Key) {
 	}
 }
 
+func (e *MapStateEntry) getNets(identities Identities, id identity.NumericIdentity) []*net.IPNet {
+	if identities == nil {
+		return nil
+	}
+	if id == identity.ReservedIdentityWorld {
+		return []*net.IPNet{
+			&net.IPNet{IP: net.IPv4zero, Mask: net.CIDRMask(0, net.IPv4len*8)},
+			&net.IPNet{IP: net.IPv6zero, Mask: net.CIDRMask(0, net.IPv6len*8)},
+		}
+	}
+	lbls := identities.GetLabels(id)
+	var nets []*net.IPNet
+	for _, lbl := range lbls {
+		if lbl.Source == labels.LabelSourceCIDR {
+			_, netIP, err := net.ParseCIDR(lbl.Key)
+			if err == nil {
+				nets = append(nets, netIP)
+			}
+		}
+	}
```

Comment:
(incidentally, given the above, I feel like there should be a bug that this code triggers but I can't quite wrap my head around what that bug would look like just yet.)

Reply:
> (incidentally, given the above, I feel like there should be a bug that this code triggers but I can't quite wrap my head around what that bug would look like just yet.)

I don't think there is a possible bug because the largest subnet in `a` will at least check against the smallest subnet in `b`.

Reply:
The concern I have is that the largest subnet in `a` will be 0.0.0.0/0, and that will match any subnet in `b`. So rather than determining whether `b` is a subset, the function always returns true.

Reply:
(discussed out-of-band, should be an easy fix to find the most specific CIDR)